src/assets/js/scope.js

```diff
@@ -13,11 +13,11 @@ class Scope {
     const {
       staffInformation: { staff: { value: isStaff = false } = {} } = {},
       accessInformation: {
-        mozilliansorg: { values: { nda: isNdaed = false } = {} } = {},
+        mozilliansorg: { values: moziiliansorgGroups = {} } = {},
```

Comment:
`values` can be `null` and that is valid profile v2. That's what was happening here.

Reply:
did you mean to type `moziiliansorgGroups` or `mozilliansorgGroups`?